Src/Plugins/Tests.Security/TestData/SecurePlaintextSecrets/ExpectedOutputs/SEC101_003.GoogleApiKey.sarif

```diff
@@ -74,6 +74,7 @@
             }
           ],
           "fingerprints": {
+            "AssetFingerprint/v1": "",
```

Comment:
>AssetFingerprint/v1": "", [](start = 13, length = 25)

Oops! A pretty clear indicate of a gap. i.e., we don't persist any information here that tries to provide a useful identifier for this exposure. like the account name. let's file an issue for this gap and keep moving. :)

Reply:
Oh, I see what we need to do! We need to add Google as an asset platform and it should be emitted from this rule as part of the fingerprint in the static validator. 

---
In reply to: [590717347](https://github.com/microsoft/sarif-pattern-matcher/pull/276#discussion_r590717347) [](ancestors = 590717347)